docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,121 +4,113 @@
+
+.. Set the certificate's Common Name (CN) to the {kib} server's hostname or FQDN. Using the server's IP address as the CN does not work.
+
+You may choose to generate a certificate and private key using {ref}/certutil.html[the {es} certutil tool]. If you already used certutil to
+generate a certificate authority (CA), you would generate a certificate/key for Kibana like so (using the `--dns` param to set the
+`subjectAltName`):
+
+[source,sh]
+--------------------------------------------------------------------------------
+bin/elasticsearch-certutil cert --ca elastic-stack-ca.p12 --name kibana --dns localhost
+--------------------------------------------------------------------------------
+
+This will generate a certificate/key contained PKCS #12 keystore named `kibana.p12`.
 
 --
 
-.. Generate a server certificate for {kib}. 
+. Enable TLS/SSL in `kibana.yml`:
 +
 --
-//TBD: Can we provide more information about how they generate the certificate? 
-//Would they be able to use something like the elasticsearch-certutil command?
-You must either set the certificate's
-`subjectAltName` to the hostname, fully-qualified domain name (FQDN), or IP
-address of the {kib} server, or set the CN to the {kib} server's hostname
-or FQDN. Using the server's IP address as the CN does not work.
+[source,yaml]
+--------------------------------------------------------------------------------
+server.ssl.enabled: true
+--------------------------------------------------------------------------------
 --
 
-.. Set the `server.ssl.enabled`, `server.ssl.key`, and `server.ssl.certificate` 
-properties in `kibana.yml`:
+. Specify your server certificate and private key in `kibana.yml`:
 +
 --
+If your certificate/key are contained in a PKCS #12 keystore, specify it like so:
+
 [source,yaml]
 --------------------------------------------------------------------------------
-server.ssl.enabled: true
-server.ssl.key: /path/to/your/server.key
-server.ssl.certificate: /path/to/your/server.crt
+server.ssl.keystore.path: "/path/to/your/keystore.p12"
+server.ssl.keystore.password: "optional decryption password"
+--------------------------------------------------------------------------------
+
+Otherwise, if your certificate/key are in PEM format, specify them like so:
+
+[source,yaml]
+--------------------------------------------------------------------------------
+server.ssl.certificate: "/path/to/your/server.crt"
+server.ssl.key: "/path/to/your/server.key"
+server.ssl.keyPassphrase: "optional decryption password"
 --------------------------------------------------------------------------------
 
 After making these changes, you must always access {kib} via HTTPS. For example, 
 https://localhost:5601.
 
-// TBD: The reference information for server.ssl.enabled says it "enables SSL for 
-// outgoing requests from the Kibana server to the browser". Do we need to 
-// reiterate here that only one side of the communications is encrypted?
```

Comment:
@lcawl that information was incorrect, I have updated the reference information for `server.ssl.enabled` to clarify. It enables SSL for inbound requests from the browser to Kibana. There are no outgoing requests from the Kibana server to the browser (there's no such thing), but traffic both ways (requests and responses) is encrypted. Let me know if you think this should be rephrased or clarified.